lib/matplotlib/cbook.py

```diff
@@ -850,11 +850,15 @@ def join(self, a, *args):
                     mapping[elem] = set_a
 
     def joined(self, a, b):
-        """Return whether *a* and *b* are members of the same set."""
+        """
+        Return whether *a* and *b* are members of the same set.
+        """
```

Comment:
This change is not necessary, in fact the original one conforms to our style https://matplotlib.org/devdocs/devel/document.html#quote-positions. Please revert.

Reply:
@timhoffm I manually changed it back in comment https://github.com/matplotlib/matplotlib/pull/29617/commits/08982e9a27d76f4a81d0a9af43e123de1928cdf2